conftest.py

```diff
@@ -263,6 +264,23 @@ def pytest_terminal_summary(terminalreporter, exitstatus, config):
                 )
                 break
 
+    main_thread = threading.main_thread()
+    if threads := [t for t in threading.enumerate() if t is not main_thread]:
+        terminalreporter.section("Leaked threads", yellow=True)
+        for idx, thread in enumerate(threads, start=1):
+            terminalreporter.write(f"{idx}: {thread}\n")
```

Comment:
I thought about throwing an excepting if leaked threads exist but we don't know who leaks threads. For example, transformers might.

Reply:
Agreed on not raising exception. Btw it is always "leaked" threads? Is it possible that some library or github action starts child threads during the test?

Reply:
> Is it possible that some library or github action starts child threads during the test?

That's also possible. Any good alternative?

Reply:
/ai suggest word that we can replace `Leaked` with.

Reply:
/ai suggest words that we can replace Leaked with.

Reply:
What about "Idle"?

Reply:
> What about "Idle"?

technically the thread can still running some task... but I don't have strong preference so any choice from the GPT's list looks also fine:)

Reply:
Remaining sounds good. let's use it